bm-persona/Common/EventTableViewCell.swift

```diff
@@ -0,0 +1,186 @@
+        eventName.rightAnchor.constraint(equalTo: contentView.layoutMarginsGuide.rightAnchor).isActive = true
+        eventName.heightAnchor.constraint(equalToConstant: 50).isActive = true
+        
+        eventTime.font = Font.regular(16)
+        eventTime.layoutMargins = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 0)
+        eventTime.leftAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leftAnchor).isActive = true
+        eventTime.rightAnchor.constraint(equalTo: contentView.layoutMarginsGuide.rightAnchor).isActive = true
+        eventTime.bottomAnchor.constraint(equalTo: contentView.layoutMarginsGuide.bottomAnchor).isActive = true
+        
+        eventCategory.font = Font.regular(12)
+        eventCategory.padding = UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5)
+        eventCategory.frame.size.height = 18
+        eventCategory.layer.masksToBounds = true
+        eventCategory.layer.cornerRadius = eventCategory.frame.height / 1.5
+        eventCategory.backgroundColor = Color.eventDefault
+        eventCategory.layoutMargins = UIEdgeInsets(top: 0, left: 0, bottom: 0, right: 15)
+        eventCategory.rightAnchor.constraint(equalTo: contentView.layoutMarginsGuide.rightAnchor).isActive = true
+        eventCategory.bottomAnchor.constraint(equalTo: contentView.layoutMarginsGuide.bottomAnchor, constant: 4).isActive = true
+    }
+    
+    func cellConfigure(entry: CalendarEntry) {
+        eventName.text = entry.name
+        
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "MM/dd/yyyy"
+        eventTime.text = dateFormatter.string(from: entry.date!)
+        
+        eventCategory.text = entry.eventType
+        
+        let entryColor = getEntryColor(entry: entry)
+        eventTaggingColor.backgroundColor = entryColor
+        eventCategory.backgroundColor = entryColor
+    }
+    
+    func getEntryColor(entry: CalendarEntry) -> UIColor {
+        switch entry.eventType {
+        case "Academic":
+            return Color.eventAcademic
+        case "Holiday":
+            return Color.eventHoliday
+        case "General":
+            return Color.eventHoliday
+        case "College of Engineering":
+            return Color.eventAcademic
+        case "College of Letters and Sciences":
+            return Color.eventAcademic
+        case "College of Chemistry":
+            return Color.eventAcademic
+        case "College of Natural Resources":
+            return Color.eventAcademic
+        default:
+            return Color.eventDefault
+        }
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+        // Configure the view for the selected state
+    }
+
+}
+
+
+extension UILabel {
```

Comment:
May want to move this to a new Utils file. Also, may want to look into Common/TagView, which does something similar.